nrrd/reader.py

```diff
@@ -15,6 +16,10 @@
 
 _NRRD_REQUIRED_FIELDS = ['dimension', 'type', 'encoding', 'sizes']
 
+# Duplicated fields are prohibited by the spec, but do occur in the wild.
+# Set True to allow duplicate fields, with a warning.
+_NRRD_ALLOW_DUPLICATE_FIELD = False
```

Comment:
I would make this a parameter to the *read* & *read_header* functions. As I said, I'm not a fan of having to edit a private member in a module, one that especially isn't documented.

I understand your apprehension not to document this because it is not supported by the NRRD specification, but I think whatever parameters we add, they should be documented. 

What you can do though, in the docstring, you can put a warning saying it is not supported. Something like this (I'm not sure if it can go in the argument list, but you can put it in the body of the docstring for sure).

```
    .. warning::
        Allowing duplicate fields with :obj:`allow_duplicate_fields` is not explicitly supported in the NRRD specification. Use this parameter at your own risk.
```

Reply:
Ok. It is no longer a private field, but I will not be adding it to `read`. This really is a special case, just like the existing chunk size parameter.